src/Components/Facility/CentralNursingStation.tsx

```diff
@@ -70,6 +76,16 @@ export default function CentralNursingStation({ facilityId }: Props) {
     "3xl": 12 / 11,
   });
 
+  const size = useBreakpoints({ "4k": 2560, default: 1920 });
+
+  useEffect(() => {
+    if (size < LARGE_SCREEN_WIDTH) {
+      setPerPageLimit(PER_PAGE_LIMIT);
+    } else {
+      setPerPageLimit(PER_PAGE_LIMIT_LARGE);
+    }
+  }, [size]);
+
```

Comment:
You don't need a `useState` nor `useEffect` when using `useBreakpoints` since you could directly set the per page limits.

```suggestion
  const perPageLimit = useBreakpoints({ "4k": 9, default: 6 });
```

Reply:
"My apologies, @rithviknishad. I became a bit confused with the useBreakpoints hook. I've made the necessary updates now